Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1

```diff
@@ -1,296 +1,122 @@
-[CmdletBinding(DefaultParameterSetName = 'None')]
-param
-(
-    [String] [Parameter(Mandatory = $true)]
-    $ConnectedServiceName,
+Trace-VstsEnteringInvocation $MyInvocation
+Import-VstsLocStrings "$PSScriptRoot\Task.json"
+
+try{
+
+    $ServiceName = Get-VstsInput -Name ServiceName -Require
+    $ServiceLocation = Get-VstsInput -Name ServiceLocation
+    $StorageAccount = Get-VstsInput -Name StorageAccount -Require
+    $CsPkg = Get-VstsInput -Name CsPkg -Require
+    $CsCfg = Get-VstsInput -Name CsCfg -Require
+    $Slot = Get-VstsInput -Name Slot -Require
+    $DeploymentLabel = Get-VstsInput -Name DeploymentLabel
+    $AppendDateTimeToLabel = Get-VstsInput -Name AppendDateTimeToLabel -Require
+    $AllowUpgrade = Get-VstsInput -Name AllowUpgrade -Require -AsBool
+    $NewServiceAdditionalArguments = Get-VstsInput -Name NewServiceAdditionalArguments
+    $NewServiceAffinityGroup = Get-VstsInput -Name NewServiceAffinityGroup
+
+    # Load all dependent files for execution
+    . $PSScriptRoot/Utility.ps1
+
+    Write-Host "Find-VstsFiles -LegacyPattern $CsCfg"
```

Comment:
Wouldn't a message like "Finding $CsCfg" be more useful to customers? `Find-VstsFiles -LegacyPattern [...]` is an internal implementation detail. Also, the function Find-VstsFiles has tracing built-in (when system.debug is true).